store/conf/attrs/zimbra-attrs.xml

```diff
@@ -9925,8 +9925,68 @@ TODO: delete them permanently from here
   <defaultCOSValue>FALSE</defaultCOSValue>
   <desc>Creates unsubscribe system folder</desc>
  </attr>
- <attr id="4004" name="zimbraPrefCalenderScaling" type="enum" value="10,15,30" cardinality="single" optionalIn="account,cos" flags="accountInherited,domainAdminModifiable" since="9.1.0">
+<attr id="4004" name="zimbraPrefCalenderScaling" type="enum" value="10,15,30" cardinality="single" optionalIn="account,cos" flags="accountInherited,domainAdminModifiable" since="9.1.0">
   <defaultCOSValue>30</defaultCOSValue>
   <desc>Default calendar resolution preference</desc>
 </attr>
+
+<attr id="4007" name="zimbraBackupConfigEnabled" type="boolean" cardinality="single" optionalIn="globalConfig,domain,cos,account" flags="accountCosDomainInherited,domainInherited" since="9.1.0">
+  <desc>Whether or not account is eligible for backup</desc>
+        If true on cos level then backup accounts for cos
+        If unset on cos level but true on domain level then backup domains account.
+        If unset on cos level, unset on domain level but true on account level then specific account will be backed up.
+        If true on cos level , unset on domain level but zimbraDomainDefaultCOSId is set to cos then accounts for that domain will be backed up.
+        If false is set on any level then that account or group of accounts will not be eligible for backup.
```

Comment:
* remove the 4th item. Instead, describe about zimbraDomainDefaultCOSId in the 1st item. Please modify it if there is any better description.
* move `</desc>` to the last.
```suggestion
  <desc>Whether or not account is eligible for backup.
        If true on cos level then backup accounts for cos. zimbraDomainDefaultCOSId is considered.
        If unset on cos level but true on domain level then backup domains account.
        If unset on cos level, unset on domain level but true on account level then specific account will be backed up.
        If false is set on any level then that account or group of accounts will not be eligible for backup.
  </desc>
```